Lib/test/test_exceptions.py

```diff
@@ -1507,6 +1507,61 @@ def f():
 
         self.assertNotIn("somethingverywronghehe", err.getvalue())
 
+    def test_name_error_bad_suggestions_do_not_trigger_for_small_names(self):
```

Comment:
Can you add a similar test for attribute errors? (The test class is also in this same file)

Reply:
Added.